api_tests/lib/ethereum.ts

```diff
@@ -147,7 +147,7 @@ export class EthereumWallet {
 
     public async sendEthTransactionTo(
         to: string,
-        data: string = "0x0",
```

Comment:
This stupid `EthereumWallet` has bitten us so many times now, on `bobtimus` as well as here in the tests.
I think we should extract what we have here into an NPM package and publish it. Basically an idiot-proof `EthereumWallet` that cannot be used wrongly :D

Reply:
hahahahaha, yes I think it's worth considering and to give back to the community.

Reply:
Created ticket here: https://github.com/coblox/bobtimus/issues/78